code/obj/item/gun/russian.dm

```diff
@@ -35,9 +35,9 @@
 	proc/fire_gun(mob/user as mob)
 		if(src.shotsLeft > 1)
 			src.shotsLeft--
-			playsound(user, "sound/weapons/Gunclick.ogg", 80, 1)
 			for(var/mob/O in AIviewers(user, null))
 				if (O.client)
+					playsound(user, "sound/weapons/Gunclick.ogg", 80, 1)
 					O.show_message("<span class='alert'>[user] points the gun at \his head. Click!</span>", 1, "<span class='alert'>Click!</span>", 2)
```

Comment:
I'm not sure this is the right fix. This'll play the sound for every single viewer. so if you have 50 people viewing you, it'll play the sound 50 times....

this whole thing should just be src.visible_message

Reply:
oh